src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Index.cshtml

```diff
@@ -344,7 +344,14 @@ else
                         @apprenticeship.CourseName
                     </td>
                     <td class="govuk-table__cell" data-label="Planned start date">
-                        @apprenticeship.PlannedStartDate.ToGdsFormatWithoutDay()
+                        @if (apprenticeship.IsOnFlexiPaymentPilot.GetValueOrDefault() && apprenticeship.ActualStartDate.HasValue)
+                        {
+                            @apprenticeship.ActualStartDate.GetValueOrDefault().ToGdsFormat();
+                        }
+                        else if(!apprenticeship.IsOnFlexiPaymentPilot.GetValueOrDefault())
```

Comment:
`apprenticeship.IsOnFlexiPaymentPilot is not true`

Reply:
we can use newish c# features in this project, pattern matching reads a lot better imho:
```


@if (apprenticeship.IsOnFlexiPaymentPilot is true && apprenticeship.ActualStartDate.HasValue)
{
    @apprenticeship.ActualStartDate.GetValueOrDefault().ToGdsFormat();
}
else if (apprenticeship.IsOnFlexiPaymentPilot is not true)
{
    @apprenticeship.PlannedStartDate.ToGdsFormatWithoutDay();
} 
```